strongbox-web-core/src/main/java/org/carlspring/strongbox/security/authentication/suppliers/JWTAuthenticationSupplier.java

```diff
@@ -53,7 +55,16 @@ public boolean supports(HttpServletRequest request)
     {
         String authHeader = request.getHeader(AUTHORIZATION_HEADER);
 
-        if (authHeader != null && authHeader.startsWith(BEARER_AUTHORIZATION_PREFIX))
+        boolean hasCookie = false;
+
+        if (request.getCookies() != null)
+        {
+            hasCookie = Arrays.stream(request.getCookies())
+                              .anyMatch(c -> c.getName()
+                                              .equals(AUTHORIZATION_COOKIE));
+        }
+
+        if ((authHeader != null && authHeader.startsWith(BEARER_AUTHORIZATION_PREFIX)) || hasCookie)
```

Comment:
`(authHeader != null && authHeader.startsWith(BEARER_AUTHORIZATION_PREFIX))` this probably better to assign into `hasHeader` variable like you've done it with `hasCookie`